### ISSUE TITLE
Problems about model parallel

### ISSUE THREAD

**Issue by Shaw95** [NVIDIA/Megatron-LM#111]
Hi, thank you for providing such an excellent framework

I met a problem when using it  to implement my own network, I found that if I train on 8 GPUs with only data parallel, the loss  converges normally.  While if I trained with 2-way MP and 4-way DP, the loss of the first epoch is OK, but afterwards the loss is going up. 

I'm just wondering what possibly causes this problem and if there are any tips for using model parallism.

Thanks a lot!
Shaw

**Comment by bugface** [comment]
Had the same problem.  Try to increase the batch size and/or lower the learning rate.
